pkg/tools/gen/testdata/jsonschema/pattern/expect.k

```diff
@@ -0,0 +1,23 @@
+"""
+This file was generated by the KCL auto-gen tool. DO NOT EDIT.
+Editing this file might prove futile when you re-run the KCL auto-gen generate command.
+"""
+import regex
+
+schema Cronjob:
+    """
+    Cronjob
+
+    Attributes
+    ----------
+    name : str, optional
+    schedule : str, optional, default is "5 0 * * *"
+    command : str, optional
+    """
+
+    name?: str
+    schedule?: str = "5 0 * * *"
+    command?: str
+
+    check:
+        regex.match(schedule, "^(\d+|\*)(/\d+)?(\s+(\d+|\*)(/\d+)?){4}$")
```

Comment:
s/`"^(\d+|\*)(/\d+)?(\s+(\d+|\*)(/\d+)?){4}$"`/`r"^(\d+|\*)(/\d+)?(\s+(\d+|\*)(/\d+)?){4}$"`. Use the raw string for the regex string.

Reply:
changed to use the raw value